src/components/mobile.css

```diff
@@ -117,7 +117,7 @@
   }
 
   .side-nav.side-nav--open {
-    max-height: 3250px; /* TODO: Magic number. Get programmatically based on content. */
+    max-height: 100vh;
```

Comment:
For content below the fold, my screen shows up empty after this change (Chrome 72 on macOS High Sierra). See below:

![Kapture 2019-03-17 at 21 12 26](https://user-images.githubusercontent.com/2608321/54493783-e0f94580-48f9-11e9-9655-d979ac576a49.gif)